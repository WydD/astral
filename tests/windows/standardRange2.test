<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query>
<window>
<description range="2" slide="1" wtype="T"/>
<source attributes="T,v,PHYID" id="S" sourceAttributes="T,v,PHYID" type="Stream"/>
</window>
</query>
<sources>
<source attributes="T,v,PHYID" file="tests/windows/S1.S" id="S" impl="FileStreamImpl" type="Stream"/>
</sources>
</astral>
