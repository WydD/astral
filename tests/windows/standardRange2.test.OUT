v,T
t=1
1,1
t=2
1,1
2,2
3,2
t=2
1,1
2,2
3,2
4,2
5,2
t=3
2,2
3,2
4,2
5,2
t=4
6,4
t=6
